pkg/rtc/mediatrack.go

```diff
@@ -651,3 +692,50 @@ func (t *MediaTrack) DebugInfo() map[string]interface{} {
 func (t *MediaTrack) Receiver() sfu.TrackReceiver {
 	return t.receiver
 }
+
+func (t *MediaTrack) GetUpConnectionScore() float64 {
+	t.statsLock.Lock()
+	defer t.statsLock.Unlock()
+	return t.connectionStatsUp.Score
+}
+
+func (t *MediaTrack) closeChan() {
+	close(t.done)
+}
+
+func (t *MediaTrack) updateStats() {
+	for {
+		select {
+		case _, ok := <-t.done:
+			if !ok {
+				return
+			}
+		case <-time.After(lastUpdateDelta):
+
+			// take lock on track access buffer
+			t.lock.Lock()
+			var stats buffer.Stats
+			if t.buffer != nil {
+				stats = t.buffer.GetStats()
+			}
+			t.lock.Unlock()
+
+			// lock the stats
+			t.statsLock.Lock()
+			t.connectionStatsUp.Curr.TotalPackets += stats.PacketCount
```

Comment:
I believe we can get the total packet count from the Reception Report.. would just need to handle wrapping. https://github.com/pion/rtcp/blob/master/reception_report.go#L22

It seems better to get both total and loss from the same data source.